src/js/tools/sentryBabelTransformerUtils.ts

```diff
@@ -0,0 +1,69 @@
+import { logger } from '@sentry/utils';
+import * as fs from 'fs';
+import * as path from 'path';
+import * as process from 'process';
+
+import type { BabelTransformer } from './vendor/metro/metroBabelTransformer';
+
+/**
+ * Saves default Babel transformer path to the project root.
+ */
+export function saveDefaultBabelTransformerPath(defaultBabelTransformerPath: string): void {
+  try {
+    fs.mkdirSync(path.join(process.cwd(), '.sentry'), { recursive: true });
+    fs.writeFileSync(getDefaultBabelTransformerPath(), defaultBabelTransformerPath);
+    logger.debug('Saved default Babel transformer path');
+  } catch (e) {
+    // eslint-disable-next-line no-console
+    console.error('[Sentry] Failed to save default Babel transformer path:', e);
+  }
+}
+
+/**
+ * Reads default Babel transformer path from the project root.
+ */
+export function readDefaultBabelTransformerPath(): string | undefined {
+  try {
+    if (fs.existsSync(getDefaultBabelTransformerPath())) {
+      return fs.readFileSync(getDefaultBabelTransformerPath()).toString();
+    }
```

Comment:
I think this pattern of fs.exists then fs.readFile should be avoided. It technically is a race condition. readFile throws anyhow if the file doesn't exist.